### PR TITLE
Add new collection with column specification

### PR DESCRIPTION
**Pull request by kubilayorcun**
# Description
Added new functionality for creating new table which corresponds to a collection in our project. Takes a string and a String arrayList as an argument, String value provides tableName and arrayList provides column names to be added to that specific table.

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/DatabaseOperations.java [hunk 1]
```diff
@@ -130,6 +171,41 @@ public ResultSet allTables() throws SQLException {
...
+
+        Statement addColumnStt = connection.createStatement();
+
+        addColumnStt.executeUpdate(addColumnQ);
+    }
+
+
+    // Deleting choosen column from table, will be added.
+    public void deleteColumnFromTable(String tableName, String columnName){
+
```
Comment (sasalx) [inline src/DatabaseOperations.java]: I assume you will add this one later.

**Comment by sasalx** [comment]
Other than my one comment, I can't see a problem.

**Comment by kubilayorcun** [comment]
I think you meant `editDataFromTable` method, and yes it will be little bit tricky but since we'll show the table data in a table form at the front-end side we'll already have the id attributes in the result set which is coming from `SELECT * FROM <tableName>`, there can be a id column of the items in the created ui component (id col can be visible or not, depends on UX). When that edit button clicked program will fetch the id attribute of the respective row (it may fetch that data by looking at source button of the action event.), so in the end we can use that specific id to edit the row data. If anything seems ambiguous or redundant we can discuss the issue further.

**Comment by sasalx** [comment]
Oh, I see. In theory, it will work but we need to work/see the front-end code. For now, it's fine.

**Comment by kubilayorcun** [comment]
I agree, also in case of a failure also i have somewhat a backup plan at hand if this approach fails to accomplish the task but for now i just wait it because solution at hand is more concise than the backup plan. So we will see as you say.

**Pull request merged**